src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt

```diff
@@ -131,10 +135,10 @@ class CreateMafVcf : CliktCommand(help = "Create gVCF and hVCF from Anchorwave M
         return gvcfVariantsByContig.keys
             .sorted() //Need to do a sort here as we need to make sure we process the chromosomes in order
```

Comment:
Comment on using sort:  How will this sort chr1, chr9, chr10?  In Biokotlin we handled this with a alphaThenNumeric sort.  I'm using it in places like this:

var sortedRecords = records.sortedWith(compareBy(SeqRangeSort.alphaThenNumberSort){ name:String -> name})

Something to consider if we need to ensure chr9 is sorted before chr10.

Reply:
Didn't know this was a thing, added it in here and something similar to when we are loading up the BED file to sort the chroms correctly.